en/sensor/pmw3901.md

```diff
@@ -71,7 +71,10 @@ The PX4 configuration that is common to all PMW3901-based boards:
 
 In addition:
 - The distance sensors should be enabled as described in their setup documentation (no flow-specific configuration is required).
-  For sensors connected by SPI or I2C (but not UAVCAN), [SENS_EN_PMW3901](../advanced_config/parameter_reference.md#SENS_EN_PMW3901) must be set `1` in order to enable the sensor driver.
+  For sensors connected by SPI or I2C (but not UAVCAN/UART), [SENS_EN_PMW3901](../advanced_config/parameter_reference.md#SENS_EN_PMW3901) must be set `1` in order to enable the sensor driver.
```

Comment:
Thanks @bys1123 !

I've done a minor subedit and I plan to merge shortly (at some point in future might make sense for us to split the docs for each sensor up. 

Anyway, one question, for the UART sensor do I STILL need to set `SENS_EN_PMW3901`? Or is setting the UART enough?
My guess is that it is, but wanted make sure.

Reply:
SENS_EN_PMW3901 only enabled SPI version PMW3901 driver, (I don't think this driver have I2C supported, needs to confirm). So, no need to enable it for UART version.

Reply:
Thanks, I'll update.